File: Techblazers-Part2/even_numbers.py
list_of_even_digit_numbers = []
for number in range(1000, 3001):
    num_str = str(number)
    for digit in num_str:
        if int(digit) % 2 != 0:
            break
    else:
        list_of_even_digit_numbers.append(number)

print(list_of_even_digit_numbers)

File: Techblazers-Part2/EvenNumbers.java
import java.util.ArrayList;

public class EvenNumbers{
   public static void main(String []args){

	ArrayList <Integer> evenNumbers = new ArrayList <>();

	for(int number = 1000; number <= 3000; number++){
		boolean isEven = true;
		int tempNumber = number;
		while(tempNumber != 0){
			int digit = tempNumber % 10;
			tempNumber = tempNumber / 10;

			if(digit % 2 != 0){
				isEven = false;
				break;
			}
		}
		
		if(isEven){
			evenNumbers.add(number);
		}
	}

	System.out.print(evenNumbers);
   }
}